reference/content-types/index.rst

```diff
@@ -0,0 +1,78 @@
+Content types
+=============
+
+As already described in :doc:`/book/creating-a-basic-website/adding-a-template`
+a template consists of multiple content types, which enable the user to manage
+content in a semantic way.
+
+The simplest template possible looks something like the this:
+
+.. code-block:: xml
+
+    <?xml version="1.0" ?>
+    <template xmlns="http://schemas.sulu.io/template/template"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
+        <key>default</key>
+
+        <view>ClientWebsiteBundle:templates:default</view>
+        <controller>SuluWebsiteBundle:Default:index</controller>
+        <cacheLifetime>2400</cacheLifetime>
+
+        <meta>
+            <title lang="en">Default</title>
+        </meta>
+
+        <properties>
+            <property name="title" type="text_line" mandatory="true">
+                <meta>
+                    <title lang="en">Title</title>
+                </meta>
+
+                <tag name="sulu.rlp.part"/>
+            </property>
+
+            <property name="url" type="resource_locator" mandatory="true">
+                <meta>
+                    <title lang="en">Resourcelocator</title>
+                </meta>
+
+                <tag name="sulu.rlp"/>
+            </property>
+        </properties>
+    </template>
+
+This chapter will describe which types you can insert within the
+``properties``-tag. Every content type in the documentation comes with an
```

Comment:
Remove the `-` should read

````
``properties`` tag
````

(not even sure if it will render when you hyphenate an inline code thing)